roles/vpn/templates/etc_openvpn_server.conf.j2

```diff
@@ -190,6 +190,7 @@ ifconfig-pool-persist ipp.txt
 ;push "redirect-gateway def1 bypass-dhcp"
 push "redirect-gateway def1"
 push "dhcp-option DNS 10.8.0.1"
+push "dhcp-option DNS 8.8.8.8"
```

Comment:
If people want to add 8.8.8.8 they typically do it on the VPN client side, right? I personally don't mind adding Google's DNS here but other people might.